data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1684 @@
+			`	He leaves without saying anything else, leading to you and the others exchanging confused glances. "Much to take in at once, it must be for him, in particular. A Heliarch he once was, after all," Tummug says. "Get to work on perfecting the footage, and readying it for release to the public, we should."`
+			`	Oobat agrees, and the two prepare to leave with the footage. "An unique experience the trip was, Captain. Hope I do that work together again, we soon will."`
+			choice
+				`	"Same here, it was different from my usual work, working a camera and all."`
+				`	"Anytime. Just, let's find some more normal people to interview, next time."`
+			`	She chuckles, and nods before leaving the room, following Tummug out of the Lunarium complex.`
+			branch heliarch
+				has "assisted heliarch"
+			`	Chiree approaches you, bringing a small box up to your hands. You pick it up, and examining it further notice it's one of the Coalition's translation devices. "One of us you now are, so capable of understanding what we say without others aiding you, you should be," she says. "Still readying ourselves for major assaults on the Heliarchs, we are. A difficult fight against them, we will have, all the way. Hoping, I am, that with your help we can count, when the time comes."`
+				goto end
+			label heliarch
+			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts, and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for I do not wish to know. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`
```

Comment:
```suggestion
			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for wish to know, I do not. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`
```